uber/templates/base.html

```diff
@@ -69,7 +69,8 @@
             {% endif %}
             {% if c.HAS_STUFF_ACCESS %}
                 {Schedule: [
-                    {'View Schedule': '../schedule/'},
+                    {% if c.ALT_SCHEDULE_URL %}{'View External Schedule': '../schedule/'},{% endif %}
```

Comment:
damn. need.... jinja2..... can solve this so easily with the new template inheritance stuff..... #soon